project/AppDependencies.scala

```diff
@@ -6,17 +6,16 @@ private object AppDependencies {
 
   val compile = Seq(
     ws,
-    "uk.gov.hmrc" %% "bootstrap-play-26"             % "1.8.0",
+    "uk.gov.hmrc" %% "bootstrap-frontend-play-27"    % "2.21.0",
     "uk.gov.hmrc" %% "simple-reactivemongo"          % "7.30.0-play-26",
```

Comment:
This is still play 26?

Reply:
Also you only need bootstrap-frontend-play-27, bootstrap-play-26 can go

Reply:
changes made